codegen/src/main/twirl/templates/JavaClient/ClientPowerApi.scala.txt

```diff
@@ -48,4 +31,4 @@ public abstract class  @{service.name}ClientPowerApi {
         throw new java.lang.UnsupportedOperationException();
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
What, the imports weren't even used?!

Reply:
Some of them were just carried over from other templates it seems